.github/workflows/build.yml

```diff
@@ -25,5 +25,15 @@ jobs:
           java-version: ${{ matrix.java }}
           cache: 'maven'
```

Comment:
the changes below are not needed, since caching is enabled at this line, see: https://github.com/actions/setup-java#caching-packages-dependencies